security.md

```diff
@@ -2,16 +2,19 @@
 
 ## Supported Versions
 
-Security fixes are provided in new releases and bugfix releases.
+Securiy advisories are published at:
+https://github.com/nim-lang/security/security/advisories?state=published
+
+Security fixes are provided in new releases and in bugfix releases.
 
 We do not backport security fixes to older releases.
 
 (Yet, Linux distributions might backport security fixes for their packages.)
 
 ## Reporting a Vulnerability
 
-Please do not report vulnerabilities via GitHub issues.
-
-If you have discovered a vulnerability, it is best to notify us about it via
+If you have discovered a vulnerability, please notify us about it via
 security@nim-lang.org in order to set up a meeting where we can discuss the next
```

Comment:
Should we instead set up a Telegram group for reports? I feel like emails can more easily get lost.

Reply:
It's fine for now; I move emails to nim-lang/security/issues so that nothing gets lost.